template.yaml

```diff
@@ -39,7 +39,7 @@ Conditions:
   SpotResponseEventMappingDisabled:
     !Or [ !Equals [ integration, !Ref Environment ], !Equals [ production, !Ref Environment ] ]
   EnableProvisionedConcurrency:
-    !Or [ !Equals [!Ref Environment, staging], !Equals [!Ref Environment, build], !Equals [!Ref Environment, integration]]
+    !Or [ !Equals [!Ref Environment, staging], !Equals [!Ref Environment, build], !Equals [!Ref Environment, integration], !Equals [!Ref Environment, production]]
```

Comment:
Can we switch to`!Not [ !Equals [!Ref Environment, dev] ]` eventually ?

Reply:
Yep, I was thinking to do this when cleaning up after. but just wanted to definitely convince myself we're doing the right thing in production this time 😅

Reply:
Cool - was thinking the same thing. Good stuff!